src/Apps/Order/Routes/__tests__/Shipping.test.tsx

```diff
@@ -98,27 +98,26 @@ describe("Shipping", () => {
       expect(config.variables.input.orderId).toBe("1234")
     })
 
+    fillAddressForm(component, validAddress)
+
```

Comment:
A buuunch of the tests in this file were passing before this PR when they shouldn't have been. Many of them because they forgot to fill in the address form before clicking submit. I haven't had time to look into why that was but it is very surprising indeed.